### DISCUSSION TITLE
What's the RAM requirement of the Stable Diffusion example?

### DISCUSSION THREAD

**Discussion by CCRcmcpe** [hpcaitech/ColossalAI#1863] [Community | Q&A]
When I set `ColossalAIStrategy` parameter `placement_policy` to `cuda`, still get OOM when I'm using the Stable Diffusion example.
RAM is 12G, VRAM 16G. Batch size = 1
What's the RAM requirement? I think 12G RAM without offloading is more than enough

Edit:
Trying with 25G RAM in Colab still no success.

**Comment by Fazziekey** [comment]
can you offer more detail, I think 12G is enough to run it, the weight of the whole model is about 2G

**Comment by vazkir** [comment]
@CCRcmcpe did you figure out what the requirements were to not get a OOM? Would love to know as I want to give it a shot as well!
